commands/project/shell/root.sh

```diff
@@ -0,0 +1,36 @@
+## Opens privileged shell on first container
+#
+# Command
+#    project:shell:root
+#    s:e:r
+#    shell-root
+
+function project:shell:root() {
+    _checkProject
```

Comment:
@sbaerlocher we should clean this up in another issue, this entire block could be reused.

Reply:
#68

Reply:
@sbaerlocher, thanks for creating the ticket! Just a quick note - when you wrap up a conversation, could you drop a quick comment about why you resolved it? It helps me keep track of what's been sorted and why. Cheers!